src/libLookup/Lookup.cpp

```diff
@@ -4015,7 +4159,8 @@ bool Lookup::Execute(const bytes& message, unsigned int offset,
       &Lookup::ProcessForwardTxn,
       &Lookup::ProcessGetDSGuardNetworkInfo,
       &Lookup::ProcessSetHistoricalDB,
-      &Lookup::ProcessGetCosigsRewardsFromSeed};
+      &Lookup::ProcessGetCosigsRewardsFromSeed,
+      &Lookup::ProcessSetMinerInfoFromSeed};
```

Comment:
Need to add `ProcessSetMinerInfoFromSeed` to exclusion list of `Lookup::ToBlockMessage`.
Otherwise the message received is currently being discarded since node is not yet synced.